src/Microsoft.TemplateEngine.Utils/TemplateParameterExtensions.cs

```diff
@@ -4,16 +4,32 @@
 #nullable enable
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using Microsoft.TemplateEngine.Abstractions;
 
 namespace Microsoft.TemplateEngine.Utils
 {
     public static class TemplateParameterExtensions
     {
+        private const char[]? _whitespaceSeparators = null;
```

Comment:
what is use case for these separators?

Reply:
This allows [fall-back splitting on white characters](https://github.com/dotnet/runtime/blob/215b39abf947da7a40b0cb137eab4bceb24ad3e3/src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs#L1618)